website/pages/api-docs/operator/autopilot.mdx

```diff
@@ -0,0 +1,214 @@
+---
+layout: api
+page_title: Autopilot - Operator - HTTP API
+sidebar_title: Autopilot
+description: |-
+  The /operator/autopilot endpoints provide tools for managing Autopilot.
+---
+
+# Autopilot Operator HTTP API
+
+The `/operator/autopilot` endpoints allow for automatic operator-friendly management of Nomad servers including cleanup of dead servers, monitoring the state of the Raft cluster, and stable server introduction.
```

Comment:
We don't have a linter for it yet (although the docs folks say one is coming), but most of our docs hard-wrap markdown lines.